server/api/routes/meeting.cjs

```diff
@@ -93,6 +94,81 @@ router.delete('/:meetingId', requireToken, isAdmin, async (req, res, next) => {
     next(err);
   }
 });
+router.post('/active/messages', requireToken, async (req, res, next) => {
+  try {
+    let correctRecip;
+    const meeting = await Meeting.findOne({
+      where: {
+        isClosed: false,
+        meetingStatus: 'confirmed',
+        [Op.or]: [{ userId: req.user.id }, { buddyId: req.user.id }],
+      },
+    });
+    if (meeting === undefined) {
```

Comment:
I believe a query that doesn't return a result shows as 'null' not 'undefined' so I would change this to be `if (!meeting)...`